lib/migrator.js

```diff
@@ -33,6 +33,8 @@ module.exports = class Migrator {
   }
 
   skipTemplatesUsedAsLayoutName(templateFilePaths) {
+    console.info(`\nChecking if any component templates are used as templates of other components using \`layoutName\``);
```

Comment:
Adding a newline at the beginning helped with making log messages more readable. I think we could spend more time writing logs that are easy to read and understand by the end user.